src/material/datepicker/datepicker-intl.ts

```diff
@@ -65,6 +65,12 @@ export class MatDatepickerIntl {
    */
   endDateLabel = 'End date';
 
+  /**
+   * A label used to indicate that the date is part of a selected comparison
+   * sub range of dates (used by screen readers).
+   */
+  comparisonDateLabel = 'Sub range';
```

Comment:
Is there any canonical phrasing here to indicate a comparison range? Is "Sub range" typical?